packages/stark-core/tsconfig-build.json

```diff
@@ -11,7 +11,9 @@
       "@angular/router": ["../../node_modules/@angular/router"],
       "rxjs/*": ["../../node_modules/rxjs/*"],
       "environments/environment": ["./src/common/environment"],
-      "@nationalbankbelgium/stark-core": ["."]
+      "@nationalbankbelgium/stark-core": ["."],
```

Comment:
Previously, `@nationalbankbelgium/stark-<package>": ["."],` was always the last mentioned path, is not it important so ? 😊

Reply:
Technically speaking, no... it is not important. What could be important is a path like this: `"*" : ["some-path"]` which indeed should be at the end because the TS compiler checks these paths sequentially from top to bottom, so the more generic paths should be at the end. 

But that is not the case here 😉 What I just did here was to keep the paths in alphabetical order...

Reply:
Okido. Thanks for the explanation 😊 
In this case, could we change the order like this ?
```json
"paths": {
      "@angular/common": ["../../node_modules/@angular/common"],
      "@angular/core": ["../../node_modules/@angular/core"],
      "@angular/router": ["../../node_modules/@angular/router"],
      "@nationalbankbelgium/stark-core": ["."],
      "core-js/*" : ["../../node_modules/core-js/*"],
      "environments/environment": ["./src/common/environment"],
      "rxjs/*": ["../../node_modules/rxjs/*"],
      "zone.js/*": ["../../node_modules/zone.js/*"]
    },
```

Otherwise I find this a bit weird if it's not sorted in alphabetical order. What do you think ? 😊

Reply:
True, I forgot to sort it... I'll do it ;)